EconomistDigitalSolutions/goberry
# Dependencies - vendor flag

TO TEST:

1. Check out branch.
2. Run ```git submodule init``` and ```git submodule update```.
3. Run ```go build```.

This branch also moves the web stuff into a small package so the main package is at the top level, drops some scattered shell scripts and brings the Makfile back.
